lib/cc/engine/analyzers/php/parser.rb

```diff
@@ -16,7 +16,7 @@ def initialize(code, filename)
           end
 
           def parse
-            runner = CommandLineRunner.new("php #{parser_path}")
+            runner = CommandLineRunner.new("php -d 'display_errors = Off' #{parser_path}")
```

Comment:
PHP duplication specs were failing for me because notices polluted the output. This is definitely setup issue but I don't know if CI has similar setup to that of mine. I can extract this commit and see how CI will react to that.